blog/2024-12-01-StartUpWeekExperience.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Saint Louis Startup Connection: Showcasing Open Source SLU’s Value to the Innovation Ecosystem
```

Comment:
value needs to be in quotes because of the colon, but I think it's probably too long anyway. Just shorten this to "Saint Louis Startup Connection" and delete the rest.